extensions/warp-ipfs/src/store/files.rs

```diff
@@ -32,26 +36,12 @@ use crate::{
 use super::{ecdh_decrypt, ecdh_encrypt, event_subscription::EventSubscription, get_keypair_did};
 
 #[derive(Clone)]
-#[allow(dead_code)]
 pub struct FileStore {
     index: Directory,
     path: Arc<RwLock<PathBuf>>,
-    modified: DateTime<Utc>,
-    index_cid: Arc<RwLock<Option<Cid>>>,
-    thumbnail_store: ThumbnailGenerator,
-
-    location_path: Option<PathBuf>,
-
-    ipfs: Ipfs,
-    constellation_tx: EventSubscription<ConstellationEventKind>,
-
-    signal: futures::channel::mpsc::UnboundedSender<()>,
-
     config: config::Config,
-
-    span: Span,
-
-    signal_guard: Arc<Notify>,
+    tx: mpsc::Sender<FileTaskCommand>,
```

Comment:
would be good to have a more descriptive name.

Reply:
> tx should be obvious that its a sending channel

That part is clear. What's not obvious is what is being sent over this channel and that makes the code less readable IMO. The names you removed where better, although I'm not sure if those still apply.

Reply:
> That part is clear. What's not obvious is what is being sent over this channel and that makes the code less readable IMO. The names you removed where better, although I'm not sure if those still apply.

If one is reading over the code, I would assume that `tx:  mpsc::Sender<FileTaskCommand>` would be noticed, which would indicate what is being sent. Even then, reading further would indicate what is being sent via channel. I dont really believe it makes it less readable though. 

> The names you removed where better, although I'm not sure if those still apply.
Those arent applicable in this case :)